src/main/java/woowa/sw/baseball/RandomBallGenerate.java

```diff
@@ -0,0 +1,26 @@
+package woowa.sw.baseball;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * RandomBall 객체를 이용한 난수 생성 class
+ * auther : sw
+ */
+public class RandomBallGenerate {
+    private final static int MAX_RANDOM_NUM = 9;
+    private final static int MIN_RANDOM_NUM = 1;
+    private final static int RANDOM_BALL_CNT = 3;
+
+    // 난수 생성
+    private int generateRandomBall() {
+        return (int) (Math.random() * MAX_RANDOM_NUM) + MIN_RANDOM_NUM;
+    }
+
+    public Set<Integer> makeThreeRandomBalls() {
+        Set<Integer> threeRandomBallSet = new LinkedHashSet<>();
+        for (int i=0; i<RANDOM_BALL_CNT; i++) threeRandomBallSet.add(new RandomBall(generateRandomBall()).getRandomNum());
```

Comment:
```suggestion
        for (int i = 0; i < RANDOM_BALL_CNT; i++) {
            threeRandomBallSet.add(new RandomBall(generateRandomBall()).getRandomNum());
        }
```
for 문 안에서 연산자 좌우로 공백을 넣어주는 것과 중괄호로 둘러싸는 것이 자바 컨벤션에 맞습니다.


그리고 RandomBalls 라는 클래스가 있는데, 여기서 List<Integer> 를 저장하는 것도 좋지만 List<RandomBall> 을 저장하는 것은 어떨까요??
어떤 다른 클래스들의 자료구조를 상태로 가지고 있는 클래스를 `일급 컬렉션`이라고 합니다.

좀더 객체지향적인 코드를 짜기 위해서 일급 컬렉션을 사용하고, 안에 있는 값을 사용할 때 값을 꺼내는 방식이 아니라 메시지를 보내서 일급 컬렉션 클래스 내부에서 로직을 실행하도록 코드를 작성하는게 좋습니다.

[일급 컬렉션 관련 블로그](https://jojoldu.tistory.com/412)